src/main/java/edu/harvard/iq/dataverse/mydata/DataRetrieverAPI.java

```diff
@@ -514,4 +521,8 @@ private JsonArrayBuilder formatSolrDocs(SolrQueryResponse solrResponse, RoleTagR
         return jsonSolrDocsArrayBuilder;
         
     }
+
+    private boolean isValid(SolrSearchResult result, DataverseRequest dataverseRequest) {
```

Comment:
I think I agree that the permission check here isn't needed (not sure why it wouldn't work), but that means DataverseRequest isn't needed as a param/ doesn't have to be created above.

Reply:
I cleaned it up.